### PR TITLE
Added Registered Symbol

### PR DESCRIPTION
**Pull request by linkfang**
For Infra-4243
Got +1 from Sharon

### CODE REVIEW

**Review by autumnfound — APPROVED** [review]

**Comment by ahus1** [comment]
Hi @linkfang, thanks for preparing the changes. 

Please have a look at the following two things: 

* README.adoc contains a section "Trademarks" - I suppose AsciiDoc should be added there as well.
* On the main page, the "AsciiDoc Working Group" used to be in one line, now it breaks into two lines. Please adjust (maybe via jumbotron_class  in config.toml) that this doesn't break at least on the main page by making is a little wider. See screenshots below. 

Thanks!

Before: 

image


After: 

image

**Comment by chrisguindon** [comment]
I created a PR to update our base theme: https://github.com/EclipseFdn/asciidoc-wg.eclipse.org/pull/76

That PR will be needed to be merged in before this one.

@linkfang Can you make sure to remove layouts/partials/jumbotron.html  from your PR. Those changes are now in our base theme. We don't need to override this template anymore.

**Comment by ahus1** [comment]
@linkfang - the other PR is now merged into main. Unfortunately this PR now has a merge conflict: please resolve them.

Please consider squashing the commits in this ticket before I merge them into main; alternatively I can squash the commits via the GitHub web UI.

**Comment by chrisguindon** [comment]
@ahus1 I think it's preferable that you squash them when you merge the PR!

**Comment by ahus1** [comment]
@chrisguindon - looking at this issue this is awaiting a review approval from you. Please leave a note here if you plan to do some work here or if you're done. 

Besides that I'm now ready to merge.

**Review by chrisguindon — APPROVED** [review]

**Comment by mojavelinux** [comment]
Is it really necessary to put the registered trademark symbol on AsciiDoc in both the title and the description of the hero? Can't we just put it in one place? If I have a choice, it would be in the description.

**Comment by chrisguindon** [comment]
@mojavelinux I will share your question with our Trademark specialist. We are currently acting under their direction for this.

File: content/_index.adoc
Comment (chrisguindon) [inline content/_index.adoc:4]: @linkfang I am looking at our jumbotron template and it appears as we are duplicating the h1 and p tag here: 
https://github.com/EclipseFdn/hugo-solstice-theme/blob/master/layouts/partials/jumbotron.html#L19

Can you remove here.

**Review by chrisguindon — CHANGES_REQUESTED** [review]

File: content/_index.adoc
Comment (linkfang) [reply content/_index.adoc]: Sure, good catch!

**Comment by chrisguindon** [comment]
> Is it really necessary to put the registered trademark symbol on AsciiDoc in both the title and the description of the hero? Can't we just put it in one place? If I have a choice, it would be in the description.

We don't need it in both places. We usually add it in the first occurrence of the word but it's fine to only keep it in the description.

@linkfang Can you make this last change please?

**Comment by ahus1** [comment]
Thank you for everyone involved here. The change is now live.

**Pull request merged**
